<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">PeakAscentLogger</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>10</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="topBarLayout" stretch="0,0,0,0,0,0,0,1,0,0,0,0,0">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="newAscentButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New ascent...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/ascent.svg</normaloff>:/icons/ascent.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newPeakButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New peak...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/peak.svg</normaloff>:/icons/peak.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newTripButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New trip...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/trip.svg</normaloff>:/icons/trip.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newHikerButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New hiker...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/hiker.svg</normaloff>:/icons/hiker.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newRegionButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New region...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/region.svg</normaloff>:/icons/region.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newRangeButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New mountain range...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/range.svg</normaloff>:/icons/range.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="newCountryButton">
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>New country...</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/country.svg</normaloff>:/icons/country.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>25</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="topBarSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ascentCounterFilteredLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Number of ascents currently shown in the table with filters applied</string>
        </property>
        <property name="text">
         <string>Shown ascents:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="ascentCounterFilteredSegmentNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>30</height>
         </size>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="ascentCounterSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="ascentCounterLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Logged ascents:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="ascentCounterSegmentNumber">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>75</width>
          <height>30</height>
         </size>
        </property>
        <property name="digitCount">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="mainAreaTabs">
      <property name="locale">
       <locale language="English" country="Europe"/>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <widget class="MainWindowTabContent" name="ascentsTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/ascent.svg</normaloff>:/icons/ascent.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Ascents</string>
       </attribute>
      </widget>
      <widget class="MainWindowTabContent" name="peaksTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/peak.svg</normaloff>:/icons/peak.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Peaks</string>
       </attribute>
      </widget>
      <widget class="MainWindowTabContent" name="tripsTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/trip.svg</normaloff>:/icons/trip.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Trips</string>
       </attribute>
      </widget>
      <widget class="MainWindowTabContent" name="hikersTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/hiker.svg</normaloff>:/icons/hiker.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Hikers</string>
       </attribute>
      </widget>
      <widget class="MainWindowTabContent" name="regionsTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/region.svg</normaloff>:/icons/region.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Regions</string>
       </attribute>
      </widget>
      <widget class="MainWindowTabContent" name="rangesTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/range.svg</normaloff>:/icons/range.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Mountain ranges</string>
       </attribute>
      </widget>
      <widget class="MainWindowTabContent" name="countriesTab" native="true">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/country.svg</normaloff>:/icons/country.svg</iconset>
       </attribute>
       <attribute name="title">
        <string>Countries</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="statisticsTab">
       <attribute name="icon">
        <iconset>
         <normaloff>:/icons/ico/statistics_multisize_square.ico</normaloff>:/icons/ico/statistics_multisize_square.ico</iconset>
       </attribute>
       <attribute name="title">
        <string>Statistics</string>
       </attribute>
       <layout class="QVBoxLayout" name="statisticsTabLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="fileMenu">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="openRecentMenu">
     <property name="title">
      <string>Open recent database</string>
     </property>
     <addaction name="separator"/>
     <addaction name="clearRecentDatabaseListAction"/>
    </widget>
    <addaction name="newDatabaseAction"/>
    <addaction name="openDatabaseAction"/>
    <addaction name="openRecentMenu"/>
    <addaction name="saveDatabaseAsAction"/>
    <addaction name="closeDatabaseAction"/>
    <addaction name="separator"/>
    <addaction name="projectSettingsAction"/>
    <addaction name="separator"/>
    <addaction name="settingsAction"/>
   </widget>
   <widget class="QMenu" name="viewMenu">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="showFiltersAction"/>
    <addaction name="separator"/>
    <addaction name="showItemStatsPanelAction"/>
    <addaction name="showAllStatsPanelsAction"/>
    <addaction name="hideAllStatsPanelsAction"/>
    <addaction name="pinStatsRangesAction"/>
    <addaction name="separator"/>
    <addaction name="autoResizeColumnsAction"/>
    <addaction name="resetColumnOrderAction"/>
    <addaction name="restoreHiddenColumnsAction"/>
    <addaction name="separator"/>
    <addaction name="addCustomColumnAction"/>
    <addaction name="separator"/>
    <addaction name="clearTableSelectionAction"/>
   </widget>
   <widget class="QMenu" name="newMenu">
    <property name="title">
     <string>New</string>
    </property>
    <addaction name="newAscentAction"/>
    <addaction name="newPeakAction"/>
    <addaction name="newTripAction"/>
    <addaction name="newHikerAction"/>
    <addaction name="newRegionAction"/>
    <addaction name="newRangeAction"/>
    <addaction name="newCountryAction"/>
   </widget>
   <widget class="QMenu" name="toolsMenu">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="findPeakLinksAction"/>
    <addaction name="relocatePhotosAction"/>
    <addaction name="exportDataAction"/>
   </widget>
   <widget class="QMenu" name="helpMenu">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="aboutAction"/>
    <addaction name="aboutQtAction"/>
   </widget>
   <addaction name="fileMenu"/>
   <addaction name="viewMenu"/>
   <addaction name="newMenu"/>
   <addaction name="toolsMenu"/>
   <addaction name="helpMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="locale">
    <locale language="English" country="Europe"/>
   </property>
  </widget>
  <action name="newDatabaseAction">
   <property name="text">
    <string>New database...</string>
   </property>
  </action>
  <action name="openDatabaseAction">
   <property name="text">
    <string>Open database...</string>
   </property>
  </action>
  <action name="clearRecentDatabaseListAction">
   <property name="text">
    <string>Clear list</string>
   </property>
  </action>
  <action name="saveDatabaseAsAction">
   <property name="text">
    <string>Save database as...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="closeDatabaseAction">
   <property name="text">
    <string>Close database</string>
   </property>
  </action>
  <action name="projectSettingsAction">
   <property name="text">
    <string>Project settings...</string>
   </property>
  </action>
  <action name="settingsAction">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="showFiltersAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show filters</string>
   </property>
  </action>
  <action name="showItemStatsPanelAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show statistics panel in this tab</string>
   </property>
   <property name="shortcut">
    <string notr="true">F7</string>
   </property>
  </action>
  <action name="showAllStatsPanelsAction">
   <property name="text">
    <string>Show statistics in all tabs</string>
   </property>
  </action>
  <action name="hideAllStatsPanelsAction">
   <property name="text">
    <string>Hide statistics in all tabs</string>
   </property>
  </action>
  <action name="pinStatsRangesAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Always use maximum axis ranges</string>
   </property>
   <property name="shortcut">
    <string notr="true">F8</string>
   </property>
  </action>
  <action name="autoResizeColumnsAction">
   <property name="text">
    <string>Auto-resize columns</string>
   </property>
  </action>
  <action name="resetColumnOrderAction">
   <property name="text">
    <string>Reset column order</string>
   </property>
  </action>
  <action name="restoreHiddenColumnsAction">
   <property name="text">
    <string>Restore hidden columns</string>
   </property>
  </action>
  <action name="addCustomColumnAction">
   <property name="text">
    <string>Add custom column...</string>
   </property>
  </action>
  <action name="clearTableSelectionAction">
   <property name="text">
    <string>Clear table selection</string>
   </property>
   <property name="shortcut">
    <string notr="true">Esc</string>
   </property>
  </action>
  <action name="newAscentAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/ascent.svg</normaloff>:/icons/ascent.svg</iconset>
   </property>
   <property name="text">
    <string>New ascent...</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+N</string>
   </property>
  </action>
  <action name="newPeakAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/peak.svg</normaloff>:/icons/peak.svg</iconset>
   </property>
   <property name="text">
    <string>New peak...</string>
   </property>
  </action>
  <action name="newTripAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/trip.svg</normaloff>:/icons/trip.svg</iconset>
   </property>
   <property name="text">
    <string>New trip...</string>
   </property>
  </action>
  <action name="newHikerAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/hiker.svg</normaloff>:/icons/hiker.svg</iconset>
   </property>
   <property name="text">
    <string>New hiker...</string>
   </property>
  </action>
  <action name="newRegionAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/region.svg</normaloff>:/icons/region.svg</iconset>
   </property>
   <property name="text">
    <string>New region...</string>
   </property>
  </action>
  <action name="newRangeAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/range.svg</normaloff>:/icons/range.svg</iconset>
   </property>
   <property name="text">
    <string>New mountain range...</string>
   </property>
  </action>
  <action name="newCountryAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/country.svg</normaloff>:/icons/country.svg</iconset>
   </property>
   <property name="text">
    <string>New country...</string>
   </property>
  </action>
  <action name="findPeakLinksAction">
   <property name="text">
    <string>Auto-fill peak links...</string>
   </property>
  </action>
  <action name="relocatePhotosAction">
   <property name="text">
    <string>Relocate photos...</string>
   </property>
  </action>
  <action name="exportDataAction">
   <property name="text">
    <string>Export data...</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset>
     <normaloff>:/icons/logo.svg</normaloff>:/icons/logo.svg</iconset>
   </property>
   <property name="text">
    <string>About PAL</string>
   </property>
  </action>
  <action name="aboutQtAction">
   <property name="icon">
    <iconset>
     <normaloff>:/qt-project.org/qmessagebox/images/qtlogo-64.png</normaloff>:/qt-project.org/qmessagebox/images/qtlogo-64.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainWindowTabContent</class>
   <extends>QWidget</extends>
   <header>src/main/main_window_tab_content.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
